lib/config.js

```diff
@@ -61,6 +62,16 @@ function loadConfig(filePath) {
         }
     }
 
+    // Recursively merge parent configurations
+    if (config.extends) {
+        var parentPath = (!isAbsolutePath(config.extends) ?
+            path.join(path.dirname(filePath), config.extends) :
+            config.extends
+        );
+
+        config = util.mergeConfigs(loadConfig(parentPath), config);
```

Comment:
I wonder if there should be a try-catch block around this? It would be nice to have an error message specify which config file cause this problem (or rather which one was the root of the problem).